WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -646,17 +646,21 @@ public static boolean isGutenbergEditorEnabled() {
         return true;
     }
 
-    public static void enableGutenbergEditor(boolean enabled) {
-        setBoolean(DeletablePrefKey.GUTENBERG_EDITOR_ENABLED, enabled);
+    public static boolean shouldAutoEnableGutenbergForTheNewPosts() {
+        return getBoolean(DeletablePrefKey.SHOULD_AUTO_ENABLE_GUTENBERG_FOR_THE_NEW_POSTS, true);
+    }
+
+    public static void autoEnableGutenbergForThewNewPosts(boolean enable) {
```

Comment:
It seems there is a typo here.

Reply:
What's about using the set/get convention for these new 2 methods? 
`setAutoEnableGutenbergForNewPosts` and `getAutoEnableGutenbergForNewPosts`

Reply:
Thanks @daniloercoli! It's fixed now.